### PR TITLE
Separated integration tests coverage threshold

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Separated integration tests coverage threshold

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Configuration Update
- [ ] Bump-up dependent library

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,7 +32,7 @@ test-unit: ## Run the unit tests
 test-integration: ## Run integration tests tests
 	@echo "Running integration tests..."
 	@echo "Reports will be written to ${ARTIFACT_DIR}"
-	python -m pytest tests/integration --cov=ols --cov-report term-missing --cov-report json:${ARTIFACT_DIR}/coverage_integration.json --junit-xml=${ARTIFACT_DIR}/junit_integration.xml
+	python -m pytest tests/integration --cov=ols --cov-report term-missing --cov-report json:${ARTIFACT_DIR}/coverage_integration.json --junit-xml=${ARTIFACT_DIR}/junit_integration.xml --cov-fail-under=50
```
Comment (onmete) [inline Makefile:35]: Docs says 60, I see 50 here.

Comment (tisnik) [reply Makefile]: aaand you are right, fixed @onmete

**Pull request merged**